### ISSUE TITLE
Accessibility: aria-live implication in web/Statusbar could cause highly annoying UX

### ISSUE THREAD

**Issue by KreerC** [owncast/owncast#4382]
See https://github.com/owncast/owncast/blob/508895fd90396c594e2b2e712ba307d89746d655/web/components/ui/Statusbar/Statusbar.tsx#L78-L79

Because `role="status"` is set on the parent container, its aria-live implication will override the below explicit `aria-live="off"`declaration for some assistive technology, updating and pulling attention every second. This could be an issue with some screen readers, magnifying applications, or switch control schemes. 

In the worst case, this would reposition the users' focused zoom areas every second, or make screen readers output "Live for x" over and over again, which of course is highly undesired. 

I suggest removing both the `role` and the `aria-live` attributes, as neither the countdown nor the view count is relevant enough to be observed automatically anyway.

**Comment by gabek** [comment]
I had thought we fixed this already with https://github.com/owncast/owncast/pull/3361/files, so it's weird hearing that the parent element is overriding the properties set on the child. It's usually the opposite.

But I'll certainly defer to the experts here, so thank you for your recommendation. Sounds like a good plan to me.

<blockquote><img src="https://avatars.githubusercontent.com/u/46684200?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/owncast/owncast/pull/3361">Remove "Live for XXX" message from aria-live area by armadi1809 · Pull Request #3361 · owncast/owncast</a></strong></div><div>This should resolve #3334
Description
This PR sets the aria-live attribute to &quot;off&quot; on the &lt;span&gt; responsible for rendering the &quot;live for XXX&quot; message to avoid spamming sc...</div></blockquote>

**Comment by gmarav05** [comment]
hey @gabek can you assign me this issue

**Comment by gabek** [comment]
Of course, thank you!

**Comment by gabek** [comment]
@gmarav05 Are you still looking at this?

**Comment by gmarav05** [comment]
yeah yeah I will solve it today.

**Comment by gmarav05** [comment]
@gabek please take a look at this

<img width="998" height="205" alt="Image" src="https://github.com/user-attachments/assets/fbec0b32-a75b-4ebe-a4f1-845d8dc5417e" />

<img width="1156" height="234" alt="Image" src="https://github.com/user-attachments/assets/21684603-c684-481b-afbf-1d31a1a3a37d" />

<img width="1628" height="312" alt="Image" src="https://github.com/user-attachments/assets/0809e1a6-356d-4c1e-813c-707b0ee709b1" />

**Comment by gabek** [comment]
Do you have the web project setup and dependencies installed? It looks like maybe you opened just a single file, but not the project and it's unable to open any other files.

Here's the doc about getting a dev environment running if you haven't had a chance to see it: https://docs.owncast.dev/development

<blockquote><img src="https://www.notion.so/images/meta/default.png" width="48" align="right"><div>Notion</div><div><strong><a href="https://www.notion.so">The AI workspace that works for you. | Notion</a></strong></div><div>A tool that connects everyday work into one space. It gives you and your teams AI tools—search, writing, note-taking—inside an all-in-one, flexible workspace.</div></blockquote>

**Comment by gmarav05** [comment]
yeah sorry now its fine i forgot i just forked everything from scrarch and i re opened editor now everything is fine. @gabek please check PR

<img width="1096" height="432" alt="Image" src="https://github.com/user-attachments/assets/bb9e70c8-72e1-4a06-8744-c380259069db" />
